Sources/Lifecycle/Lifecycle.swift

```diff
@@ -190,7 +190,7 @@ public struct ServiceLifecycle {
     ///
     /// Designed for composition purposes, mainly for frameworks that need to offer both top-level start/stop functionality and composition into larger systems.
     /// In other words, should not be used outside the context of building an Application framework.
-    public let underlying: ComponentLifecycle
+    private let underlying: ComponentLifecycle
```

Comment:
Do we maybe need to update the comment? @yim-lee @tomerd

Reply:
good catch